man/meson.build

```diff
@@ -199,6 +199,5 @@ if git.found()
                 command : ['sh', '-c',
                            'cd @0@ && '.format(meson.build_root()) +
                            'python3 @0@/tools/make-man-rules.py `git ls-files ":/man/*.xml"` >t && '.format(meson.source_root()) +
-                           'mv t @0@/rules/meson.build'.format(meson.current_source_dir())],
-                depend_files : custom_entities_ent)
```

Comment:
You drop the dependency on `custom_entities_ent`, which is quite important… Do you think it's not possible to keep it?